internal/github/user_test.go

```diff
@@ -0,0 +1,216 @@
+package github_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/google/go-github/v47/github"
+	"github.com/migueleliasweb/go-github-mock/src/mock"
+	gh "github.com/qbarrand/gitstream/internal/github"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserHelperImpl_GetUser(t *testing.T) {
+
+	const (
+		owner       = "owner"
+		repo        = "repo"
+		authorEmail = "yonibettan@gmail.com"
+	)
+
+	var (
+		authorName  = "Yoni Bettan"
+		authorLogin = "ybettan"
+	)
```

Comment:
I think we should remove any personal data from that test.

Reply:
Fixed.